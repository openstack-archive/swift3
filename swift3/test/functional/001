#!/bin/bash

# GET Service

. ./common

S3USER=tester

_s3_put /abcdefghijklmnopqrstuvwxyz_01234
_s3_put /5ABCDEFGHIJKLMNOPQRSTUVWXYZ.5-6789

_s3_get / -D - | _filter_curl xml
