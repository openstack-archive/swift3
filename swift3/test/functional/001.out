QA output created by 001
> s3curl --id tester -- -X PUT http://SWIFT_HOST/abcdefghijklmnopqrstuvwxyz_01234... 200
> s3curl --id tester -- -X PUT http://SWIFT_HOST/5ABCDEFGHIJKLMNOPQRSTUVWXYZ.5-6789... 200
> s3curl --id tester -- -X GET -D - http://SWIFT_HOST/... 200
HTTP/1.1 200 OK
Content-Length: LENGTH
Content-Type: application/xml
Date: DATE
X-Trans-Id: TXID

<?xml version='1.0' encoding='UTF-8'?>
<ListAllMyBucketsResult xmlns="http://s3.amazonaws.com/doc/2006-03-01/">
  <Owner>
    <ID>TESTER</ID>
    <DisplayName>TESTER</DisplayName>
  </Owner>
  <Buckets>
    <Bucket>
      <Name>5ABCDEFGHIJKLMNOPQRSTUVWXYZ.5-6789</Name>
      <CreationDate>DATE</CreationDate>
    </Bucket>
    <Bucket>
      <Name>abcdefghijklmnopqrstuvwxyz_01234</Name>
      <CreationDate>DATE</CreationDate>
    </Bucket>
  </Buckets>
</ListAllMyBucketsResult>
